docs/rackhd/configuration.rst

```diff
@@ -215,3 +220,144 @@ choice. Verify the certificate by restarting on-http and visiting
 **Note:** For information about OpenSSL, see the `OpenSSL documentation`_.
 
 .. _OpenSSL documentation: https://www.openssl.org/docs/
+
+
+Setting up HTTP/HTTPS endpoint
+------------------------------
+
+This section describes how to setup HTTP/HTTPS endpoints in RackHD.
+A endpoint is an instance of HTTP or HTTPS server that serves a group of APIs. User can
+choose to enable authentication or enable HTTPs for each endpoint.
+
+There are currently two group of APIs defined in RackHD:
```

Comment:
I think it sounds a little better to say "There are currently two API groups defined in RackHD"